recipes/indexlr.cpp

```diff
@@ -29,55 +29,67 @@ const static size_t INITIAL_OUTPUT_STREAM_SIZE = 100;
 const static size_t QUEUE_SIZE = 64;
 const static size_t MAX_THREADS = 5;
 const static size_t DEFAULT_THREADS = MAX_THREADS;
+const static size_t MAX_QUALITY = 30;
 
 static void
 print_error_msg(const std::string& msg)
 {
   std::cerr << PROGNAME << ' ' << VERSION << ": " << msg << std::endl;
 }
 
+static void
+print_warning_msg(const std::string& msg)
```

Comment:
I think this is equal to `btllib::check_warning(true, msg)`. If so, use that function instead for consistency.